src/flick/flick_auth/controllers/authenticate_controller.py

```diff
@@ -21,8 +21,9 @@ def __init__(self, request, data, serializer):
         self._request = request
         self._data = data
         self._serializer = serializer
-        self._first_name = self._data.get("first_name")
-        self._last_name = self._data.get("last_name")
+        self._name = data.get("name")
+        self._username = data.get("username")
+        self._email = data.get("email")
```

Comment:
do we need email now too? :o

Reply:
@haiyingweng wanted us to keep track of emails!